src/PowerShellEditorServices.Protocol/LanguageServer/ServerCapabilities.cs

```diff
@@ -42,6 +42,8 @@ public class ServerCapabilities
         public ExecuteCommandOptions ExecuteCommandProvider { get; set; }
 
         public object Experimental { get; set; }
+
+        public bool FoldingRangeProvider { get; set; } = false;
```

Comment:
`bool` defaults to false so the `= false` shouldn't be needed I think. Since `bool` is a non-nullable type

Reply:
IMO, in this case, being explicit makes it more obvious what's going on.  I don't feel there's any  maintenance cost with leaving this in.

Reply:
Works for me 👍